java/feature-layer-show-attributes/src/main/java/com/esri/arcgisruntime/sample/featurelayershowattributes/MainActivity.java

```diff
@@ -83,42 +83,28 @@ public boolean onSingleTapConfirmed(MotionEvent e) {
           mCallout.dismiss();
         }
         // get the point that was clicked and convert it to a point in map coordinates
-        final Point clickPoint = mMapView
-            .screenToLocation(new android.graphics.Point(Math.round(e.getX()), Math.round(e.getY())));
+        final Point clickPoint = new Point(Math.round(e.getX()),
```

Comment:
let's try to stick with "screenPoint" for clarity, since we use that name in other samples.

Can also be on one line

Reply:
no problem!